packages/dev/docs/pages/contribute.mdx

```diff
@@ -16,3 +16,132 @@ keywords: [react spectrum contributions, bug reports, feature requests]
 ---
 
 # Contribute
+
+Thanks for choosing to contribute! We look forward to improving web applications together. Here you will find information on how propose bug fixes, suggest improvements and how to build react-spectrum yourself.
+
+## Better together
+We believe in sharing our process, results and journey with you all. The core team and all external contributors follow the same process in order maintain a high quality codebase.
+
+## Code of conduct
+We adhere to the Adobe [code of conduct](https://github.com/adobe-private/react-spectrum-v3/blob/CODE_OF_CONDUCT.md) and by participating, you are expected to uphold this code. Please report unacceptable behavior to Grp-opensourceoffice@adobe.com.
+
+## Reporting issues
+### Bugs
+We use [GitHub issues](https://github.com/adobe-private/react-spectrum-v3/issues) to track work and log bugs. Please check existing issues before filing anything new. We do our best to respond to issues within a few days. If you would like to contribute a fix, please let us know on the issue.
+
+The best way to reduce back and forth on a bug is to add code for a storybook example exhibiting the issue along with steps to reproduce it. If you would like to work on a bugfix yourself, make sure an issue exists first.
```

Comment:
I think we don't need it to be storybook specifically? Something like a code sandbox link would also work. Just need to be a minimal reproduction.

```suggestion
The best way to reduce back and forth on a bug is provide a small code example exhibiting the issue along with steps to reproduce it. If you would like to work on a bugfix yourself, make sure an issue exists first.
```